zerver/views/webhooks/trello/card_actions.py

```diff
@@ -0,0 +1,197 @@
+from typing import Dict, Tuple, Any
+from datetime import datetime
+from .exceptions import UnknownUpdateCardAction
+from .templates import TRELLO_SUBJECT_TEMPLATE, TRELLO_MESSAGE_TEMPLATE
+
+SUPPORTED_CARD_ACTIONS = [
+    'updateCard',
+    'createCard',
+    'addLabelToCard',
+    'removeLabelFromCard',
+    'addMemberToCard',
+    'removeMemberFromCard',
+    'addAttachmentToCard',
+    'addChecklistToCard',
+    'commentCard'
+]
+
+CREATE = 'createCard'
+CHANGE_LIST = 'changeList'
+CHANGE_NAME = 'changeName'
+ARCHIVE = 'archiveCard'
+REOPEN = 'reopenCard'
+SET_DUE_DATE = 'setDueDate'
+CHANGE_DUE_DATE = 'changeDueDate'
+REMOVE_DUE_DATE = 'removeDueDate'
+ADD_LABEL = 'addLabelToCard'
+REMOVE_LABEL = 'removeLabelFromCard'
+ADD_MEMBER = 'addMemberToCard'
+REMOVE_MEMBER = 'removeMemberFromCard'
+ADD_ATTACHMENT = 'addAttachmentToCard'
+ADD_CHECKLIST = 'addChecklistToCard'
+COMMENT = 'commentCard'
+
+TRELLO_CARD_URL_TEMPLATE = '[{card_name}]({card_url})'
+
+ACTIONS_TO_MESSAGE_MAPPER = {
+    CREATE: 'created {card_url_template}',
+    CHANGE_LIST: 'moved {card_url_template} from {old_list} to {new_list}',
+    CHANGE_NAME: 'renamed the card from "{old_name}" to {card_url_template}',
+    ARCHIVE: 'archived {card_url_template}',
+    REOPEN: 'reopened {card_url_template}',
+    SET_DUE_DATE: 'set due date to {due_date} on {card_url_template}',
+    CHANGE_DUE_DATE: 'changed due date to {due_date} on {card_url_template}',
+    REMOVE_DUE_DATE: 'removed due date from {card_url_template}',
+    ADD_LABEL: 'added a {color} label with \"{text}\" to {card_url_template}',
+    REMOVE_LABEL: 'removed a {color} label with \"{text}\" from {card_url_template}',
+    ADD_MEMBER: 'added {member_name} to {card_url_template}',
+    REMOVE_MEMBER: 'removed {member_name} from {card_url_template}',
+    ADD_ATTACHMENT: 'added [{attachment_name}]({attachment_url}) to {card_url_template}',
+    ADD_CHECKLIST: 'added {checklist_name} checklist to {card_url_template}',
+    COMMENT: 'commented {card_url_template}'
```

Comment:
If {card_url_template} is displayed to the user as the name of the card, then the message format here should be: 'commented on {card_url_template}' .